x-pack/plugins/canvas/canvas_plugin_src/renderers/dropdown_filter/index.tsx

```diff
@@ -29,11 +30,18 @@ export const dropdownFilter: RendererFactory<Config> = () => ({
   reuseDomNode: true,
   height: 50,
   render(domNode, config, handlers) {
-    const { column, choices } = config;
+    const filter = handlers.getFilter();
     let value = '%%CANVAS_MATCH_ALL%%';
-    if (handlers.getFilter() !== '') {
-      const filterAST = fromExpression(handlers.getFilter());
+
+    if (filter !== '') {
+      const filterAST = fromExpression(filter);
       value = get(filterAST, 'chain[0].arguments.value[0]');
+
+      const filterGroup = get(filterAST, 'chain[0].arguments.filterGroup[0]');
```

Comment:
This is unfortunately becoming a common pattern, where expression args need to be synced with the filter expression at render time. Not sure how to abstract it though ☹️